app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -0,0 +1,302 @@
+
+If you are in any doubt about meeting the 3-month requirement, please email us for advice before planning your move.
+
+### Subject eligibility  
+
+The international relocation payment (IRP) has been limited to languages and physics because non-UK teachers play an important part in teaching these priority subjects. The Department for Education is piloting the IRP to encourage more non-UK teachers and trainees to bring their skills, experience and potential to English schools.
+
+### Payment
+
+Eligibility requirement checks can only be completed after you’ve arrived in England. Therefore, the international relocation payment (IRP) cannot be made before you take up your teacher training place or teaching job.
+
+In order to be paid the IRP, you must have a UK bank account (this can be digital). If you are a teacher or a salaried teacher trainee, you must also have a [UK National Insurance number](https://www.gov.uk/apply-national-insurance-number). You can apply for this once you arrive in England, as long as your visa gives you the right to work and you have a confirmed job.  
+
+You will receive a single payment from the Department for Education around the end of your first term of training or employment. You will be paid as and when your eligibility requirements have been checked – we cannot guarantee an exact payment date.  
+
+You will not have to justify any expenses or give us receipts or any other evidence of the relocation costs you have incurred.  
+
+The Department for Education reserves the right to withhold payment if you are no longer studying on an eligible course or employed in an eligible job on the payment date.
+
+Once you have received the international relocation payment, you will not be asked to pay it back, even if you leave England, your training course or your job as a teacher.
+
+### Tax
+
+If you are on a fee-paying teacher training course, you will not have to pay tax on the international relocation payment.  
+
+If you are a teacher, or a trainee on a salaried course, the Department for Education will pay the UK tax owed on the international relocation payment to ensure you receive the full £10,000, as long as your overall earnings do not exceed £50,270 a year in 2023/24. You may have to pay higher rate [UK Income Tax](https://www.gov.uk/browse/tax/income-tax) and [UK National Insurance](https://www.gov.uk/national-insurance) on any earnings which take you over £50,270.
+
+### Part-time teachers and trainees
+
+Part-time teachers and trainees will receive the international relocation payment in full, as long as they meet all the eligibility criteria.
+
+### Other types of financial support from the UK government  
+
+Teachers and newly qualified trainees who meet the eligibility requirements can also apply for the following payments in addition to the international relocation payment (IRP):
```

Comment:
```suggestion
Teachers and recently qualified trainees who meet the eligibility requirements can also apply for the following payments in addition to the international relocation payment (IRP):
```
just to move away from 'newly qualified teacher' as we don't say this any more?

Reply:
good point